src/main/resources/db/migration/all/20230725101601__add_is_parole_to_placement_requests.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE placement_requests ADD COLUMN is_parole BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Minor point but setting a default in a migration can sometimes lead to unexpected behaviour further down the line if it's missed on an INSERT statement and the default value is inserted instead (which really was just supposed to be a `set this value for all existing rows` at the point of migration).  Prefer:

```
ALTER TABLE placement_requests ADD COLUMN is_parole BOOLEAN;
UPDATE placement_requests SET is_parole = FALSE;
ALTER TABLE placement_requests ALTER COLUMN is_parole SET NOT NULL;
```

Reply:
Good call 👍